webapp/src/actions/createPullRequest.ts

```diff
@@ -71,16 +72,18 @@ export default async function sendPullRequest(
     }
 
     const nowIso = new Date().toISOString().replace(/:/g, '').split('.')[0];
-    const branchName = 'lyra-translate-' + nowIso;
+    const uuidSnippet = randomUUID().substring(5);
```

Comment:
A UUIDv4 without the first 5 characters will contain:

1. The last 3 characters of time-low: 12 bits entropy
2. time-mid: 16 bits entropy
3. time-high-and-version: 12 bits entropy (4 of the bits statically encode that the version is 4)
4. clock-seq-and-reserved: 6 bits entropy (2 bits are used to statically encode the variant `0b10`)
5. clock-seq-low: 8 bits entropy
6. and node: 48 bits entropy

https://www.rfc-editor.org/rfc/rfc4122.txt

That sums to a  whopping 102 bits of entropy which is much higher than the previous timestamp provided and certainly enough to avoid collisions.

This is just a review note, no action is required.